fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -46,11 +47,30 @@ def self.install_wwdr_certificate
       Dir.chdir('/tmp') do
         url = 'https://developer.apple.com/certificationauthority/AppleWWDRCA.cer'
         filename = File.basename(url)
-        `curl -O #{url} && security import #{filename} -k login.keychain`
+        keychain = wwdr_keychain
+        keychain.prepend("-k ") unless keychain.empty?
+        `curl -O #{url} && security import #{filename} #{keychain}`
         UI.user_error!("Could not install WWDR certificate") unless $?.success?
       end
     end
 
+    def self.wwdr_keychain
+      priority = [
+        "security list-keychains -d user",
+        "security default-keychain -d user"
+      ]
+      for command in priority
+        keychains = Helper.backticks(command, print: $verbose).split("\n")
+        unless keychains.empty?
+          # Select first keychain name from returned keychains list
+          keychain = keychains[0].strip.tr('"', '').split(File::SEPARATOR)[-1]
+          break
+        end
+        keychain = ""
```

Comment:
Regarding the rubocop failures, I think this one is accurate:

```ruby
lib/fastlane_core/cert_checker.rb:66:11: W: Useless assignment to variable - keychain. Did you mean keychains?
          keychain = keychains[0].strip.tr('"', '').split(File::SEPARATOR)[-1]
          ^^^^^^^^
```
because `keychain` is always reassigned to "" here.